client/components/admin_form/index.coffee

```diff
@@ -30,3 +30,7 @@ module.exports = class AdminEditView extends Backbone.View
 
   ignoreUnsaved: ->
     window.onbeforeunload = ->
+
+module.exports.init = ->
+  new AdminEditView
+    el: $('body')
```

Comment:
@kanaabe @craigspaeth  - I just updated the routes so that edit will return back to the edit form.  Previously it was rendering the response to the curations_edit route, but the route is broken and doesn't give the correct URL.  I was having trouble figuring out how to change the route, but replaced it with string interpolation and it goes to the correct place.